compliance_checker/tests/resources.py

```diff
@@ -16,7 +16,7 @@ def get_filename(path):
 
 
 def generate_dataset(cdl_path, nc_path):
-    subprocess.call(["ncgen", "-o", nc_path, cdl_path])
+    subprocess.call(["ncgen", "-4", "-o", nc_path, cdl_path])
```

Comment:
We can either pin out netcdf-c to <4.8.0 or use `-4` everywhere.

Reply:
I'm in favor of keeping this more explicit - especially since there are some odd Windows path issues with libnetcdf-4.8.0 currently.

Reply:
> I'm in favor of keeping this more explicit

What do you mean? Pin to `<4.8.0` for now?